.jazzy.yaml

```diff
@@ -5,9 +5,9 @@ objc: true
 sdk: iphonesimulator
 module: Purchases
 umbrella_header: Purchases/Public/Purchases.h
-module_version: 3.6.0-SNAPSHOT
+module_version: 3.6.0
 github_url: https://github.com/revenuecat/purchases-ios
-github_file_prefix: https://github.com/revenuecat/purchases-ios/tree/3.6.0-SNAPSHOT
+github_file_prefix: https://github.com/revenuecat/purchases-ios/tree/3.6.0
```

Comment:
These versions should be 3.6.0-beta-2. The only one that should be `3.6.0` is the `CFBundleShortVersionString`